src/fitnesse/responders/run/SuiteResponder.java

```diff
@@ -295,7 +295,12 @@ protected String getRerunPageName() {
     PageCrawler pageCrawler = page.getPageCrawler();
     WikiPagePath fullPath = pageCrawler.getFullPath();
     String fullPathName = PathParser.render(fullPath);
-    return "RerunLastFailures_"+fullPathName.replace(".","-");
+    if (fullPathName.startsWith("RerunLastFailures_")) {
+      String newFullPathName = fullPathName.replace(".", "-");
```

Comment:
I think it makes more sense to to the replace on line 297 to remove the duplicated code from both branches